src/pubsub/mod_pubsub_db_mnesia.erl

```diff
@@ -10,6 +10,7 @@
 
 -include("pubsub.hrl").
 -include("jlib.hrl").
+-include_lib("stdlib/include/qlc.hrl").
```

Comment:
Why do we need it?

Reply:
Without this compiler complains about `qlc:q/1` function with the following warning:

>qlc:q/1 called, but "qlc.hrl" not included

Reply:
Interesting. :)